.eslintrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
         'no-undef': 'off',
         'react-hooks/rules-of-hooks': 'error',
         'react-hooks/exhaustive-deps': 'warn',
```

Comment:
Side note (non blocking):
I would also set this rule to "error" since this can be critical, specially for a library like this one.

Reply:
Good point – lets do that as well...

I wish there would be a bit more fine-grained control over this, we only have the option to disable the entire rule where we need to. Would be awesome if we could tell eslint to just ignore one specific variable used in a hook.